README.md

```diff
@@ -40,12 +44,44 @@ Now, build the project and the project and create the container image:
 ./mvnw clean package spring-boot:build-image
 ```
 
-### Deploy to Kubernetes
+### Deploy Application to Kubernetes
 
 ```
-kubectl apply -f kubernetes/
+kubectl apply -f kubernetes/app
```

Comment:
This didn't work for me (the 2 yaml files are still in the root of the /kubernetes folder)